src/replication.c

```diff
@@ -1349,6 +1349,29 @@ void removeRDBUsedToSyncReplicas(void) {
     }
 }
 
+/* Close the repldbfd and reclaim the page cache if the client hold
+ * the last reference to replication DB */
+void closeRepldbfd(client *myself) {
+    listNode *ln;
+    listIter li;
+    int shouldInvalidateCache = 1;
```

Comment:
our variables are snake_case (for the most part).
but also, IIRC we now call it "reclaim", not "invalidate".
how about just calling it `int reclaim`?